tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -68,37 +69,45 @@ Verify GPU Operator Deployment  # robocop: disable
     # ...   nvidia-operator-validator DS
 
 Verify That Prometheus Image Is A CPaaS Built Image
-    [Tags]    Sanity   
+    [Tags]    Sanity
     ...     Tier1
-    ...     ODS-734    
+    ...     ODS-734
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=prometheus-
     Verify Container Image    redhat-ods-monitoring    ${pod}    prometheus
     ...    "registry.redhat.io/openshift4/ose-prometheus"
     Verify Container Image    redhat-ods-monitoring    ${pod}    oauth-proxy
     ...    "registry.redhat.io/openshift4/ose-oauth-proxy:v4.8"
 
 Verify That Grafana Image Is A Red Hat Built Image
-    [Tags]    Sanity    
+    [Tags]    Sanity
     ...     Tier1
-    ...     ODS-736    
+    ...     ODS-736
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=grafana-
     Verify Container Image    redhat-ods-monitoring    ${pod}    grafana
     ...    "registry.redhat.io/rhel8/grafana:7"
     Verify Container Image    redhat-ods-monitoring    ${pod}    auth-proxy
     ...    "registry.redhat.io/openshift4/ose-oauth-proxy:v4.8"
 
 Verify That Blackbox-exporter Image Is A CPaaS Built Image
-    [Tags]    Sanity    
+    [Tags]    Sanity
     ...     Tier1
-    ...     ODS-735    
+    ...     ODS-735
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=blackbox-exporter-
     Verify Container Image    redhat-ods-monitoring    ${pod}    blackbox-exporter
     ...    "quay.io/integreatly/prometheus-blackbox-exporter:v0.19.0"
 
 Verify That Alert Manager Image Is A CPaaS Built Image
-    [Tags]    Sanity    
+    [Tags]    Sanity
     ...     Tier1
-    ...     ODS-733    
+    ...     ODS-733
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=prometheus-
     Verify Container Image    redhat-ods-monitoring    ${pod}    alertmanager
     ...    "registry.redhat.io/openshift4/ose-prometheus-alertmanager"
+
+Verify Versioning Each Release
+    [Documentation]    Verify versioning each release
```

Comment:
could the doc be more specific? what "verify versioning" means?

Reply:
It means Checking Build Version of Rhods, will update it.

Reply:
You can rename the test to `Verify RHODS Release Version Number` and add it to` Tier1`